mobile/src/views/MyICardView.jsx

```diff
@@ -17,6 +17,9 @@ const MyICard = ({navigation}) => {
   if (user.verify) finalStatus = 'inactive';
   if (user.isaf_status) finalStatus = 'active';
 
+  if (!user.isaf_status && !user.verify && user.verification_image)
+    finalStatus = 'verifying account status';
```

Comment:
According to the figma [0] i think this should be "verifying account". Also, I think you need to edit `components/shared/ICardPage.jsx` to add update the UI to match the other screens.

[0] https://www.figma.com/file/0asBQf8PggPTx6DDUs9Edm/ICard?node-id=0%3A1&t=tDgVNiCdEagW98vC-0

Reply:
I've updated the UI to match the others and changed the text name.
Demo:
![image](https://user-images.githubusercontent.com/83478026/203470934-b2f9f755-a480-41da-be74-05ae1c7c7db3.png)

Reply:
One more thing, I think we have an outdated yellow in the themes file [0]. We should probably update this to be the same as the figma.

[0] https://github.com/UofA-Blueprint/ICard/blob/develop/mobile/src/utilites/Theme.js

Reply:
I believe it should be `#FAD233`

Reply:
@yismailuofa, updated the hex colour now:
<img width="325" alt="image" src="https://user-images.githubusercontent.com/83478026/203639607-83575954-f449-42b9-bc53-6c1296d9f79d.png">